### PR TITLE
fixed korean long date format

### PR DESCRIPTION
**Pull request by bigtone1284**
fixed korean long date format.

### CODE REVIEW

File: src/locale/ko.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export default moment.defineLocale('ko', {
     longDateFormat : {
         LT : 'A h시 m분',
         LTS : 'A h시 m분 s초',
-        L : 'YYYY.MM.DD',
+        L : 'YYYYMM-DD',
```
Comment (mj1856) [inline src/locale/ko.js]: This doesn't seem right

**Comment by mj1856** [comment]
Unit tests are failing

**Comment by mj1856** [comment]
The PR looks fine now, but I'm not sure if this change is correct.  CLDR has the following for Korea:

```
<dateFormats>
	<dateFormatLength type="full">
		<dateFormat>
			<pattern>U년 MMM d일 EEEE</pattern>
		</dateFormat>
	</dateFormatLength>
	<dateFormatLength type="long">
		<dateFormat>
			<pattern>U년 MMM d일</pattern>
		</dateFormat>
	</dateFormatLength>
	<dateFormatLength type="medium">
		<dateFormat>
			<pattern>y. M. d.</pattern>
		</dateFormat>
	</dateFormatLength>
	<dateFormatLength type="short">
		<dateFormat>
			<pattern>y. M. d.</pattern>
		</dateFormat>
	</dateFormatLength>
</dateFormats>
```

@jeeeyul @kyungw00k @imappak Are we sure we want to make this change?  We would be deviating from CLDR.  Are they wrong?

**Comment by kyungw00k** [comment]
@jeeeyul @imappak @say8425 
Honestly, I have no idea which documents we should follow. 
But OS shows `y. M. d` instead of 'y-M-d', which follows CLDR.
I think this `dateFormat` concerns about `written documents` not `widely used in computing`.

**Comment by ichernev** [comment]
@kyungw00k what is more commonly used in news sites and other popular media?

**Comment by ichernev** [comment]
@jeeeyul @kyungw00k @imappak any update on this? What do other popular websites use.

**Comment by kyungw00k** [comment]
@ichernev I think someone already do research and do work in https://github.com/moment/moment/pull/3608 Looks good to me!

**Comment by maggiepint** [comment]
dupe #3608

**Pull request closed without merging**